src/social-network-provider/twitter.com/ui/fetch.ts

```diff
@@ -0,0 +1,155 @@
+import {
+    bioCard,
+    fromPostSelectorsSelectPostContentString,
+    postPopupSelector,
+    postsContentSelectors,
+    postsRootSelector,
+    postsSelectors,
+    selfInfoSelectors,
+} from '../utils/selector'
+import { MutationObserverWatcher } from '@holoflows/kit'
+import Services from '../../../extension/service'
+import { PersonIdentifier } from '../../../database/type'
+import { host } from '../index'
+import { getEmptyPostInfo, SocialNetworkUI } from '../../../social-network/ui'
+import { deconstructPayload } from '../../../utils/type-transform/Payload'
+import { regexMatch, timeout } from '../../../utils/utils'
+import { hasPostPopup } from '../utils/status'
+import { equal, notEmpty, notNullable } from '../../../utils/assert'
+
+export const resolveLastRecognizedIdentity = (self: SocialNetworkUI) => {
```

Comment:
We should call it resolveLastRecognizedIdentityTwitter to avoid accidentally import at someone else.